client/state/selectors/is-calypsoify-gutenberg-enabled.js

```diff
@@ -15,6 +22,14 @@ export const isCalypsoifyGutenbergEnabled = ( state, siteId ) => {
 		return false;
 	}
 
+	if ( isEnabled( 'jetpack/gutenframe' ) ) {
+		if (
+			versionCompare( get( getSiteOptions( state, siteId ), 'jetpack_version', 0 ), '7.3', '>=' )
```

Comment:
If we were to combine the two conditionals, would the `versionCompare` call be executed in cases where `isEnabled( 'jetpack/gutenframe' )` returns false?

Reply:
It would short-circuit before that. Even if that wasn't the case, I don't see a scenario in which running `versionCompare` in a combined check would cause any trouble. 🤔

Reply:
They _could_ be combined, but I see no advantage (where this seems a little more readable to me).